### PR TITLE
ETC.tsv Lines 17344-17444

### PR DESCRIPTION
**Pull request by ttgmichael**
Please check if I mistranslated anything. It's hard making names from Korean xD

There are certain lines where I'm not sure what the meaning was. So it'll be helpful to check those lines too. :)

### CODE REVIEW

**Comment by Sunaries** [comment]
I'll check this tomorrow. Meanwhile, I see that the names of the class masters don't have spaces. Don't they have spaces in the original file?

**Comment by ttgmichael** [comment]
@Sunaries not sure, I'm just following the format from the Korean text.

File: ETC.tsv [hunk 1]
```diff
@@ -15502,7 +15502,7 @@ ETC_20151001_015503	Gloomy Green Apparition
 ETC_20151001_015504	The Panto Totem has been destroyed.{nl}Move on to the next location!
 ETC_20151001_015505	Defeat Clymen!
 ETC_20151001_015506	Clymen has been defeated.{nl}Move on to the next location!
-ETC_20151001_015507	Protect the Paladin Barrier from the Pantos!
+ETC_20151001_015507	Protect the Romuva Barrier from the Pantos!
```
Comment (Sunaries) [inline ETC.tsv:15505]: Mistake by IMC? Original says Paladin.

File: ETC.tsv [hunk 2]
```diff
@@ -16576,8 +16576,8 @@ ETC_20151102_016576
 ETC_20151102_016577
 ETC_20151102_016578
 ETC_20151102_016579
-ETC_20151102_016580
-ETC_20151102_016581
+ETC_20151102_016580	Namott
+ETC_20151102_016581	Namott Holy Water Effect: Demon monsters will not appear.
```
Comment (Sunaries) [inline ETC.tsv]: Remove the period.

File: ETC.tsv (hunk 1 above)
Comment (ttgmichael) [inline ETC.tsv:15505]: I think so, there is no Paladin Barrier? ^^

Comment (Sunaries) [inline ETC.tsv:15505]: @imcgames '판토들로부터 팔라딘 결계를  지켜내세요!' It says 'Paladin Barrier', but is this supposed to be 'Romuva Barrier'? Typo?

**Comment by imcgames** [comment]
@Sunaries Yes, it seems Typo by planning team.

**Comment by Sunaries** [comment]
@imcgames Ah, I see. Thanks for the confirmation. :)

:+1:  good to merge

**Comment by imcgames** [comment]
:+1:

**Pull request merged**